tests/vmi_multus_test.go

```diff
@@ -995,6 +1004,46 @@ var _ = Describe("[Serial]Macvtap", func() {
 			})
 		})
 	})
+
+	Context("VMI migration", func() {
+		var macvtapVMI *v1.VirtualMachineInstance
+
+		cleanupVMI := func(vmi *v1.VirtualMachineInstance) error {
```

Comment:
Why don't you use - `deleteAllVMIsAndBlockUntilTheyAreGone`?

Reply:
Well ... I could. But I feel that does too much (it iterates all namespaces and removes all the vms in them).

I can use it if you prefer.

Reply:
You used it in the previous tests of the same feature so lets at leas be consistent:)

Reply:
Done.